students/JaeKim/session03/slicing.py

```diff
@@ -0,0 +1,64 @@
+def exchange_first_last(seq):
+    '''
+    first_letter = seq[0:1]
+    last_letter = seq[-1]
```

Comment:
Every exercise in this program should use slicing - how can you modify this to do so?